### PR TITLE
Itp cleanup 2b

### PR DESCRIPTION
**Pull request by aehyvari**

### CODE REVIEW

File: src/proof/InterpolationContext.cc [hunk 1]
```diff
@@ -12,8 +12,9 @@
 
 InterpolationContext::InterpolationContext(SMTConfig & c, Theory & th, TermMapper & termMapper, Proof const & t,
                                            PartitionManager & pmanager, int n)
-                                           : logic(th.getLogic()), pmanager(pmanager), config(c), proof_graph {
-    new ProofGraph(c, th, termMapper, t, pmanager, n)} {
+                                           : logic(th.getLogic()), pmanager(pmanager), config(c)
+                                           , proof_graph {new ProofGraph(c, th, termMapper, t, pmanager, n)}
+{
```
Comment (blishko) [inline src/proof/InterpolationContext.cc:17]: This is now formatted in #425.

File: src/proof/InterpolationContext.h [hunk 2]
```diff
@@ -23,14 +23,11 @@ class InterpolationContext {
     std::unique_ptr<ProofGraph> proof_graph;
 public:
     InterpolationContext(SMTConfig & c, Theory & th, TermMapper & termMapper, Proof const & t,
-                         PartitionManager& pmanager , int n = -1 );
+                         PartitionManager& pmanager, int n = -1 );
```
Comment (blishko) [inline src/proof/InterpolationContext.h:26]: Formatted now in #425.

File: src/proof/InterpolationContext.h [hunk 3]
```diff
@@ -23,14 +23,11 @@ class InterpolationContext {
...
     ~InterpolationContext();
 
     void printProofDotty();
 
-    // Create interpolants with each A consisting of the specified partitions
-    void getInterpolants(const std::vector<vec<int> > & partitions, vec<PTRef> & interpolants);
-
```
Comment (blishko) [inline src/proof/InterpolationContext.h:33]: I need to check this method, maybe this was the one HiFrog was using.
In that case, I need to bring back its implementation; or design a different interface.

**Comment by aehyvari** [comment]
Closing this as it has already been addressed in #450 .

**Pull request closed without merging**